content/en/integrations/guide/aws-manual-setup.md

```diff
@@ -109,17 +113,14 @@ For more details, see the [How to use an external ID when granting access to you
 
```

Comment:
^ This is for the line above. This shouldn't be there for access key govcloud/china setup since there is no external ID:

`For more details, see the [How to use an external ID when granting access to your AWS resources to a third party][1] AWS documentation.`